mypyc/primitives/registry.py

```diff
@@ -35,13 +35,21 @@
 optimized implementations of all ops.
 """
 
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, NamedTuple
 
 from mypyc.ir.ops import (
     OpDescription, EmitterInterface, EmitCallback, StealsDescription, short_name
 )
 from mypyc.ir.rtypes import RType,  bool_rprimitive
 
+# TODO: comment, we don't need error_kind since C functions are all ERR_MAGIC
```

Comment:
We'll want to have C functions that are not ERR_MAGIC. Some things will never return an error, for example.